packages/block-library/src/query/editor.scss

```diff
@@ -43,3 +43,12 @@
 		max-height: none;
 	}
 }
+
+.block-library-query-toolspanel__filters {
+	.components-form-token-field__help {
```

Comment:
I know we should avoid specific styling this way, but there is no way currently to affect the `help` styles in form token field.

Reply:
Thanks for bringing it up though, I'll add it to the list in #39358.